src/components/Articles.jsx

```diff
@@ -1,11 +1,26 @@
 import ArticleCard from './ArticleCard.jsx'
+import { Link, Routes, Route } from "react-router-dom";
+import TopicArticles from './TopicArticles.jsx'
+
 
 function Articles({ articles }) {
 
     return (
         <>
             <h1 className="articlesAll">All Articles</h1>
-            <br />
+            <div className="topicContainer">
+                <h3 className="topicHeading">Topics</h3>
+                <nav>
+                    <ul className="topicNav">
+                        <li><Link className="topicButt" to="/articles/topics/coding">Coding</Link></li>
+                        <li><Link className="topicButt" to="/articles/topics/football">Football</Link></li>
+                        <li><Link className="topicButt" to="/articles/topics/cooking">Cooking</Link></li>
```

Comment:
This topic navigation is brilliant! I would add this onto the `TopicArticles.jsx` component as well so that a user can freely navigate between each topic after they've selected one! 💡